AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -707,6 +711,13 @@ NS_ASSUME_NONNULL_END
 @property (atomic, assign)           BOOL accessibilityViewIsModal;
 @property (atomic, assign)           BOOL shouldGroupAccessibilityChildren;
 
+@property (nonatomic) UIAccessibilityNavigationStyle accessibilityNavigationStyle NS_AVAILABLE_IOS(8_0);
+#if TARGET_OS_TV
+@property(nullable, nonatomic, copy) NSArray *accessibilityHeaderElements;
```

Comment:
Should we control if this property exists by the #if TARGET_OS_TV or by adding UIKIT_AVAILABLE_TVOS_ONLY(9_0); to the declaration?